app/components/Activity/InvoiceModal.js

```diff
@@ -55,7 +55,7 @@ const InvoiceModal = ({
             <section className={styles.amount}>
               <h1>
                 <Value
-                  value={invoice.value}
+                  value={invoice.settled ? invoice.finalAmount : invoice.value}
```

Comment:
I think the `settled?` check here is redundant.

Can this be simplified to `value={invoice.finalAmount}` since `amt_paid` being set implies that the invoice has been settled.

`invoice.finalAmount` will be either the amount requested, or if a payment has already been made then it will be `amt_paid`.